tests/modules/test_securitydata.py

```diff
@@ -919,9 +925,11 @@ def test_stream_file_by_md5_raises_file_not_found_error_when_search_returns_empt
         file_event_search.text = "{}"
         file_event_client = mocker.MagicMock(spec=FileEventClient)
         file_event_client.search.return_value = file_event_search
-        with pytest.raises(Py42Error):
+        with pytest.raises(Py42ChecksumNotFoundError) as e:
             security_module.stream_file_by_md5("md5hash")
 
+        assert u"No files found with MD5 checksum" in e.value.args[0]
+
     def test_stream_file_by_md5_raises_file_not_found_error_when_file_location_returns_empty_response(
```

Comment:
hmmm... looking closer, are the resto f the tests in this module also supposed to testing for `Py42ChecksumNotFoundError`? They are all still checking the base class

Reply:
that's a different error that is raised under a different circumstance which actually _is_ just a `Py42Error`.
It's debatable whether or not that could also be made into a more specific error too, but I'd rather not have that scope creep into this PR.